TLCGen.PluggedInItems/TLCGen.Generators.CCOL/CodeGeneration/Functionality/FileIngrepenCodeGenerator.cs

```diff
@@ -481,9 +510,13 @@ public override string GetCode(ControllerModel c, CCOLCodeTypeEnum type, string
                         sb.AppendLine($"{ts}/* File ingreep {fm.Naam} */");
                         sb.AppendLine();
 
+                        //string tbz, trij, tafv, hfile;
                         foreach (var fd in fm.FileDetectoren)
                         {
-                            sb.AppendLine($"{ts}FileMeldingV2({_dpf}{fd.Detector}, {_tpf}{_tbz}{fd.Detector}, {_tpf}{_trij}{fd.Detector}, {_tpf}{_tafv}{fd.Detector}, {_hpf}{_hfile}{fd.Detector});");
+                            if (fd.IngreepNaamPerLus)
```

Comment:
Zie ook boven, voorkeur voor een one-liner met bv. coalescing operator (dwz "voorw ? otp1 : opt2"). Levert iets minder code, en daarmee in de toekomst ook beter beheer(s)baar want een wijziging kan op één plek.